healthcare/api-client/v1/hl7v2/hl7v2_messages_test.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import sys
+import uuid
+
+from gcp_devrel.testing import eventually_consistent
+
+# Add datasets for bootstrapping datasets for testing
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'datasets')) # noqa
+import datasets
+import hl7v2_stores
+import hl7v2_messages
+
+cloud_region = 'us-central1'
+project_id = os.environ['GOOGLE_CLOUD_PROJECT']
+
+dataset_id = 'test_dataset_{}'.format(uuid.uuid4())
+hl7v2_store_id = 'test_hl7v2_store-{}'.format(uuid.uuid4())
+hl7v2_message_file = 'resources/hl7-sample-ingest.json'
+label_key = 'PROCESSED'
+label_value = 'TRUE'
+
+
+@pytest.fixture(scope='module')
+def test_dataset():
```

Comment:
For these fixtures, can you also add retries like we did in datasets? It will make the tests more robust.

Reply:
Done.